autorest/codegen/__init__.py

```diff
@@ -135,12 +139,37 @@ def _build_package_dependency() -> Dict[str, str]:
             "dependency_msrest": "msrest>=0.6.21",
         }
 
+    @staticmethod
+    def _build_security(yaml_data: Dict[str, Any]) -> Dict[str, Any]:
+        security_yaml = yaml_data.get("security")
+        security: Dict[str, Any] = dict()
+        if security_yaml and security_yaml.get("authenticationRequired"):
```

Comment:
nit: if you make `security_yaml = yaml_data.get("security", {})`, you can just do `if security_yaml.get("authenticationRequired")` on this line

Reply:
good nit!